CMakeLists.txt

```diff
@@ -239,6 +252,7 @@ print_config_options()
 
 message("${PROJECT_NAME} ${PROJECT_VERSION}
 
+   version string | ${VERSION_FULL_STRING}
```

Comment:
`PROJECT_VERSION` will be always set, even if git describe doesn't work it is either set with `openage_version` file or to 0.0 if not found. So we show the full version string at the end of `./configure` as a more precise feedback.